scripts/rnnlm/validate_word_features.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
```

Comment:
I did notice this wile looking through that random initialization speedup PR -- do we really want to use python3 in general?